lte/cloud/go/services/subscriberdb/servicers/protected/lookup_test.go

```diff
@@ -21,7 +21,7 @@ import (
 
 	"magma/lte/cloud/go/services/subscriberdb"
 	"magma/lte/cloud/go/services/subscriberdb/protos"
-	"magma/lte/cloud/go/services/subscriberdb/servicers"
+	protected_servicers "magma/lte/cloud/go/services/subscriberdb/servicers/protected"
```

Comment:
Access servicers members using `servicers` package name
`protected_servicers` to be used only in case of conflicts